support-frontend/app/services/AuthenticationService.scala

```diff
@@ -27,53 +27,41 @@ object AccessCredentials {
   }
   case class Token(tokenText: String) extends AccessCredentials
 }
-case class IdMinimalUser(id: String, displayName: Option[String])
-case class AuthenticatedIdUser(credentials: AccessCredentials, minimalUser: IdMinimalUser)
 
 class AsyncAuthenticationService(identityPlayAuthService: IdentityPlayAuthService)(implicit ec: ExecutionContext) {
 
-  import AsyncAuthenticationService._
-
-  def authenticateUser(requestHeader: RequestHeader): Future[AuthenticatedIdUser] =
+  def tryAuthenticateUser(requestHeader: RequestHeader): Future[Option[User]] =
     identityPlayAuthService.getUserFromRequest(requestHeader)
-      .map { case (credentials, user) => buildAuthenticatedUser(credentials, user) }
+      .map { case (_, user) => user }
```

Comment:
we now keep the whole `User` for reuse in the controllers, rather than throwing most of it away